.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": [["@babel/preset-env", { "targets": { "node": "current" } }]]
```

Comment:
Is this here to allow jest to run test code as expected? Curious as to what Jest doesn't support in any of the features included here with the appropriate node runtime.

Reply:
> Is this here to allow jest to run test code as expected? 

That is correct.

> Curious as to what Jest doesn't support in any of the features included here with the appropriate node runtime.

Yes, that would be interesting to see - but I've tested it with "12" it works as expected - I'm not sure if we should pin it down to a specific version